server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/mailet/filter/JMAPFilteringExtension.java

```diff
@@ -112,7 +112,7 @@ public void defineRulesForRecipient1(List<Rule.Condition> conditions) {
                 .map(condition -> Rule.builder()
                     .id(Rule.Id.of(String.valueOf(counter.incrementAndGet())))
                     .name(String.valueOf(counter.incrementAndGet()))
-                    .condition(condition)
+                    .conditions(Arrays.asList(condition))
```

Comment:
`Arrays.asList` => `ImmutableList.of()` 

Guava is your friend, immutable collections is the good.